salt/metalk8s/kubernetes/apiserver/installed.sls

```diff
@@ -153,72 +93,14 @@ Create kube-apiserver Pod manifest:
           - path: {{ htpasswd_path }}
             type: File
             name: htpasswd
-{%- if pillar.metalk8s.api_server.keepalived.enabled %}
-          - path: /etc/keepalived
-            name: keepalived-config
-{%- endif %}
         sidecars:
```

Comment:
Maybe you should remove this key since it's empty

Reply:
Done